### PR TITLE
feat(elements): Add `backup_code` verification strategy

### PR DESCRIPTION
**Pull request by tmilewski**
## Description

Adds `backup_code` verification strategy

```tsx
<SignIn.Step name='choose-strategy'>
  <SignIn.SupportedStrategy name='backup_code'>Use a backup code</SignIn.SupportedStrategy>
<SignIn.Step>
```

```tsx
<SignIn.Step name='verifications'>
  <SignIn.Strategy name='backup_code'>
    <Clerk.Field name="backup_code">
      <Clerk.Label>Code:</Clerk.Label>
      <Clerk.Input />
      <Clerk.FieldError />
    </Clerk.Field>

    <Clerk.Action submit>Continue</Clerk.Action>
  </SignIn.Strategy>
<SignIn.Step>
```

Fixes SDK-1852

## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

File: packages/elements/src/internals/machines/sign-in/verification.machine.ts [hunk 1]
```diff
@@ -168,14 +169,17 @@ const SignInVerificationMachine = setup({
       },
     ),
     setConsoleError: ({ event }) => {
-      if (process.env.NODE_ENV === 'development') {
-        assertActorEventError(event);
+      if (process.env.NODE_ENV !== 'development') {
```
Comment (tmilewski) [inline packages/elements/src/internals/machines/sign-in/verification.machine.ts:172]: Removed `throw` as it puts users into a completely irrecoverable state even if `FieldError` or `GlobalError` are present.

File: packages/elements/src/react/common/form/index.tsx [hunk 2]
```diff
@@ -337,8 +337,13 @@ const useInput = ({
       type: 'text',
       spellCheck: false,
     };
-  }
-  if (type === 'password' && shouldValidatePassword) {
+  } else if (type === 'backup_code') {
```
Comment (tmilewski) [inline packages/elements/src/react/common/form/index.tsx:340]: The `code` name has a number of predefined assumptions which don't work for backup codes. 

The `backup_code` name circumvents those assumptions while still maintaining the required field name of `code`.

**Comment by tmilewski** [comment]
!snapshot

**Comment by clerk-cookie** [comment]
Hey @tmilewski - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/chrome-extension | 1.1.1-snapshot.v64cb67f |
| @clerk/clerk-js | 5.7.3-snapshot.v64cb67f |
| @clerk/elements | 0.10.0-snapshot.v64cb67f |
| @clerk/clerk-expo | 1.2.4-snapshot.v64cb67f |
| gatsby-plugin-clerk | 5.0.0-beta.45 |
| @clerk/ui | 0.1.5-snapshot.v64cb67f |

Tip: Use the snippet copy button below to quickly install the required packages.
`@clerk/chrome-extension`
```sh
npm i @clerk/chrome-extension@1.1.1-snapshot.v64cb67f --save-exact
```
`@clerk/clerk-js`
```sh
npm i @clerk/clerk-js@5.7.3-snapshot.v64cb67f --save-exact
```
`@clerk/elements`
```sh
npm i @clerk/elements@0.10.0-snapshot.v64cb67f --save-exact
```
`@clerk/clerk-expo`
```sh
npm i @clerk/clerk-expo@1.2.4-snapshot.v64cb67f --save-exact
```
`gatsby-plugin-clerk`
```sh
npm i gatsby-plugin-clerk@5.0.0-beta.45 --save-exact
```
`@clerk/ui`
```sh
npm i @clerk/ui@0.1.5-snapshot.v64cb67f --save-exact
```

**Review by panteliselef — APPROVED** [review]

**Pull request merged**